graphs/frequent_pattern_graph_miner.py

```diff
@@ -100,7 +100,7 @@ def get_support(cluster :Dict[int,Dict[str,List[str]]])-> List[float]:
     '''
     Returns support
     >>> get_support({5: {'11111': ['ab', 'ac', 'df', 'bd', 'bc']}, 4: {'11101': ['ef', 'eg', 'de', 'fg'], '11011': ['cd']}, 3: {'11001': ['ad'], '10101': ['dg']}, 2: {'10010': ['dh', 'bh'], '11000': ['be'], '10100': ['gh'], '10001': ['ce']}, 1: {'00100': ['fh', 'eh'], '10000': ['hi']}})

```

Comment:
Lines in Python should be 88 characters max.  Lines that can not be read in GitHub without left/right scrolling (127 characters) need to be wrapped for readability.